.travis.yml

```diff
@@ -31,7 +31,7 @@ addons:
 install:
   - mkdir -p ~/bin/ && ln -s /usr/bin/swig3.0 ~/bin/swig && export PATH=~/bin/:$PATH
   - pip3 install -r ./.travis_pip_reqs.txt
-  - pip3 install .[ipopt,dlib,pyswarm,pymc3,cmaes]
+  - pip3 install .[ipopt,dlib,pyswarm,pymc3,cmaes,scipy]
```

Comment:
this is not needed -- scipy is a dependency everywhere already

Reply:
Done